src/plugins/lwhmediaplay/lwhmediaplay.js

```diff
@@ -0,0 +1,223 @@
+
+/**
+ * Media play plugin - Automatically start video or audio if so desired.
+ * The media tag must then have the property "autostart" as in
+ * 	<video width="800" height="600" autostart>
+ * ATTN: using "autoplay" will start the video immediately when the presentation is loaded!
+ *
+ * If the video has the class "noconsole", the video will not be shown in the console,
+ * but be replaced by either the alternative image given in the video tag or a text with the video name.
+ *
+ * The property "transition" will play the (usually) audio when the previous step is left, thus allowing 
+ * a transition sound (like a swoosh or such).
+ *
+ * If media clips have different volumes, this can be levelled by using the attribute "data-volume"
+ * and a value between 0.0 (silent) and 1.0 (default), e.g.:
+ * 	<video width="800" height="600" autostart data-volume="0.8"> will turn the volume down by 20%.
+ * ATTN: Putting the volume up is not possible, if one clip is too low, all the others must be silenced!
+ *
+ * 2017-08-27	LWH		created
+ * 2018-01-25	LWH		audio added
+ *
+ * Copyright 2017..18	Harald Martin (LWH), www.lwh-brainware.de
+ * Released under the MIT license.
+ */
+(function ( document, window ) {
+    'use strict';
+    //	property and class names
+    const	prop_autostart	= "autostart";
+    const	class_noconsole	= "noconsole";
+    const	prop_transition = "transition";
+    const	prop_volume 	= "data-volume";
```

Comment:
DRY...

Reply:
I know about DRY, but this doesn't really achieve that. This just means you repeat one string instead of another. Since the actual strings are what actually matter, and most of them are standard DOM and aren't variables, it's more readable to use the ones that everybody knows, and not some strings you invented for yourself.

Reply:
As these strings are used _outside_ of the JS as well, I like to place them somewhere easy to find and not spread all over the code. I'll check about standards, though.